app/src/main/kotlin/com/hippo/ehviewer/ui/screen/GalleryDetailContent.kt

```diff
@@ -771,7 +771,7 @@ private fun List<GalleryTagGroup>.artistTag() = find { (ns, _) -> ns == TagNames
 
 context(Context)
 @Composable
-private fun GalleryDetail.collectPreviewItems() = rememberInVM {
+private fun GalleryDetail.collectPreviewItems() = rememberInVM(previewList) {
```

Comment:
This does not work cuz this will leave composition when refreshing (gallery detail becomes null), and old state will be stored in state map and be read when this enter composition again.